### ISSUE TITLE
[Feature] VS Code extension- Link automated test with an Azure DevOps test case

### ISSUE THREAD

**Issue by NikkTod** [microsoft/playwright#23527]
Our team is using Playwright (VSCode + NodeJS) and we would really like to have the possibility of starting an automated tests from within the Azure DevOps Test Plans. 
But in order to be able to do this, first step is to link (associate) the automated tests to the tests created inside AzureDevOps.

Here is a link that describes how to do this when people use Visual Studio + .Net core -> https://learn.microsoft.com/en-us/azure/devops/test/associate-automated-test-with-test-case?view=azure-devops

![image](https://github.com/microsoft/playwright/assets/94455079/59b51878-e757-4326-a086-6702cc117507)

I was wondering if this linking is possible to be implemented inside the VS code playwright extention? 

Our main goal is then once tests are linked we could actually build release pipeline and will be able to start particular tests directlly from Test Plan inside Azure Devops, here is a link with details -> https://learn.microsoft.com/en-us/azure/devops/test/run-automated-tests-from-test-hub?view=azure-devops

**Comment by aslushnikov** [comment]
> I was wondering if this linking is possible to be implemented inside the VS code playwright extention?

@NikkTod There's nothing specific for Azure Devops Test Plan in Playwright VSCode extension, but we have this feature request opened: https://github.com/microsoft/playwright/issues/14839

There also might be some other extension that already implements this. I'd recommend asking on the ADO Test Plan community channels - maybe they can give some better pointers.

I'll close this in favor of https://github.com/microsoft/playwright/issues/14839 - please feel free to upvote that one to bump it's priority and track progress!

**Comment by barbatrucNL** [comment]
@aslushnikov I don't understand why this ticket has been closed referencing ticket [14839](https://github.com/microsoft/playwright/issues/14839)
The title of that ticket seems a bit misleading. As far is limited to Reporting, adding testresults to Testplans. We implemented that solution provided by @alexneo2003  So our scheduled build pipeline adds results to Azure Test Plans. However, it can NOT be used to manually trigger specific automated tests from testcases within Azure Testplans as was requested above, here in tickjet 23527, by @NikkTod 

We played around with what we found and ended with this pipeline:
![image](https://github.com/user-attachments/assets/af8bc8b6-851f-4417-a0d7-9e4f96ec4c9a)

However, it doesn't find any associated tests, which ofcourse is true as we are not able to associate testcases from within VS Code / Playwrigh testrunner to Azure Test Plans. I read on Microsoft learn pages that 
"[Tests that use other test frameworks such as ... cannot be associated with a test case](https://learn.microsoft.com/en-us/azure/devops/test/associate-automated-test-with-test-case?view=azure-devops#test-types)"
After reading the documentation for [VSTest@2 - Visual Studio Test v2 task](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/vstest-v2?view=azure-pipelines&viewFallbackFrom=azure-devops) I guess Microsoft simply didn't work out anything to enable a trigger from Azure Testplan testcases to run tests automated with Playwright.
